shapeshift/web
# feat: ledger instructions popover

## Description

Does what it says on the box - makes it clear to the user that developer mode needs to be enabled in Ledger desktop by linking to Ledger support. Note, the copy is temporary here.

## Pull Request Type


## Issue (if applicable)

N/A

## Risk



None, isolated to Ledger under flag

## Testing



- Hover over the i next to THORChain
- Ensure we link to the right docs

### Engineering



- ☝🏽 

### Operations



- ☝🏽 

## Screenshots (if applicable)


<img width="547" alt="image" src="https://github.com/shapeshift/web/assets/17035424/3bb2bc05-7a3e-44a1-9d7e-4dd6df7e5974">
